install.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
 if [[ -z $OPOSSUM_HEADLESS_SETUP ]]; then
+    BOOST_INSTALLED=$(dpkg-query -W --showformat='${Status}\n' libboost1.67-dev|grep "install ok installed")
```

Comment:
If boost is not found, there is still visible output of dpkg-query

Reply:
2>/dev/null